crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -1304,6 +1305,51 @@ impl<'w, 's> SystemParamFetch<'w, 's> for SystemChangeTickState {
     }
 }
 
+#[derive(Debug)]
+pub struct SystemName {
+    name: Cow<'static, str>,
+}
+
+impl SystemName {
+    pub fn name(&self) -> &Cow<'static, str> {
+        &self.name
+    }
+}
+
+impl<'a> SystemParam for SystemName {
+    type Fetch = SystemNameState;
+}
+
+// SAFETY: Only reads World entities
+unsafe impl ReadOnlySystemParamFetch for SystemNameState {}
+
+/// The [`SystemParamState`] of [`Entities`].
```

Comment:
?

Isn't this the Param for `SystemName`?

Reply:
Ah whoops, some of this was copied from some other SystemParam stuff and wasn't paying enough attention.